src/NeonCore.ts

```diff
@@ -451,20 +461,18 @@ class NeonCore {
    * @returns If the action was undone.
    */
   undo (pageURI: string): Promise<boolean> {
```

Comment:
I refactored `undo` and `redo` while fixing the undo stack because I realized that we weren't actually resolving the promise in some conditions. The code works functionally the same as before.